rct229/rulesets/ashrae9012019/section5/section5rule38.py

```diff
@@ -1,163 +1,51 @@
-    get_zone_conditioning_category_dict,
-)
-from rct229.schema.config import ureg
-from rct229.utils.assertions import getattr_
 from rct229.utils.jsonpath_utils import find_all
-from rct229.utils.pint_utils import ZERO, CalcQ
-from rct229.utils.std_comparisons import std_equal
-
-TARGET_AIR_LEAKAGE_COEFF = 0.6 * ureg("cfm / foot**2")
-TOTAL_AIR_LEAKAGE_COEFF = 0.112
-
-MANUAL_CHECK_MSG = "The building total air leakage rate is not equal to the required proposed design air leakage rate at 75Pa with a Conversion Factor of 0.112 as per section G3.1.1.4. and Measured air leakage rate is not entered for all conditioned and semi-heated zones. Verify the proposed air leakage rate is modeled correctly."
 
 
-class Section5Rule38(RuleDefinitionListIndexedBase):
+class Section5Rule38(PartialRuleDefinition):
     """Rule 38 of ASHRAE 90.1-2019 Appendix G Section 5 (Envelope)"""
 
     def __init__(self):
         super(Section5Rule38, self).__init__(
             rmrs_used=produce_ruleset_model_instance(
-                USER=False, BASELINE_0=False, PROPOSED=True
+                USER=False, BASELINE_0=True, PROPOSED=False
             ),
-            required_fields={
-                "$": ["weather"],
-                "weather": ["climate_zone"],
-            },
-            each_rule=Section5Rule38.BuildingRule(),
-            index_rmr=PROPOSED,
             id="5-38",
-            description="The proposed air leakage rate of the building envelope (I75Pa) at a fixed building pressure differential of 0.3 in. of water shall be 0.6 cfm/ft2 for buildings providing verification in accordance with Section 5.9.1.2. The air leakage rate of the building envelope shall be converted to appropriate units for the simulation program using one of the methods in Section G3.1.1.4. Exceptions: When whole-building air leakage testing, in accordance with Section 5.4.3.1.1, is specified during design and completed after construction, the proposed design air leakage rate of the building envelope shall be as measured.",
+            description="It is acceptable to use either an annual average ground temperature or monthly average ground temperatures for calculation of heat loss through below-grade walls and basement floors.",
             ruleset_section_title="Envelope",
-            standard_section="Section G3.1-5(b) Building Envelope Modeling Requirements for the Proposed design",
-            is_primary_rule=True,
-            list_path="ruleset_model_descriptions[0].buildings[*]",
-            data_items={"climate_zone": (PROPOSED, "weather/climate_zone")},
-            manual_check_required_msg=MANUAL_CHECK_MSG,
+            standard_section="Section G3.1-14(b) Building Envelope Modeling Requirements for the Proposed design and Baseline",
+            is_primary_rule=False,
+            required_fields={
+                "$": ["weather", "ruleset_model_descriptions"],
```

Comment:
We do not need to include `ruleset_model_descriptions` in the `required_fields` since in the latest schema, this field is [required ](https://github.com/open229/ruleset-model-description-schema/blob/main/docs229/ASHRAE229.schema.json#L76)by schema.

Reply:
Thank you for the clarification!